src/content/upgrades/shard-chains/index.md

```diff
@@ -51,36 +51,6 @@ Rollups are a "layer 2" technology that exists today. They allow dapps to bundle
 [More on rollups](/developers/docs/scaling/layer-2-rollups/)
 </InfoBanner>
 
-## Shard chains version 2: code execution {#code-execution}
```

Comment:
I'd prefer to get more input here before removing this section. ([see comment](https://discord.com/channels/595666850260713488/595701195843174434/932822345821859891) from @ralexstokes in R&D discord that implementing execution shard has not been abandoned)

Reply:
Will add this back in and can be apart of a separate issue if we want. Thanks for digging that up, good context to have.